debian/changelog

```diff
@@ -0,0 +1,5 @@
+mu (0.1-1) unstable; urgency=low
```

Comment:
No worries, thanks for your help @asb, I'll see if I can find some time to have a look again and try to figure it out.